specification/app/resource-manager/Microsoft.App/preview/2022-11-01-preview/CommonDefinitions.json

```diff
@@ -1063,6 +1063,216 @@
           "readOnly": true
         }
       }
+    },
+    "ContainerAppsJobContainer": {
+      "description": "Container App container definition",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/BaseContainer"
+        }
+      ],
+      "properties": {
+        "probes": {
+          "description": "List of probes for the container.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerAppsJobProbe"
+          },
+          "x-ms-identifiers": [
+            "type"
+          ]
+        }
+      }
+    },
+    "ContainerAppsJobProbe": {
+      "description": "Probe describes a health check to be performed against a container to determine whether it is alive or ready to receive traffic.",
+      "type": "object",
+      "properties": {
+        "failureThreshold": {
+          "description": "Minimum consecutive failures for the probe to be considered failed after having succeeded. Defaults to 3. Minimum value is 1. Maximum value is 10.",
+          "type": "integer",
+          "format": "int32"
+        },
+        "grpc": {
```

Comment:
We can remove GRPC probe at least for V1?

Reply:
yes. Let's remove it.

Reply:
@trajkobal, @vinisoto reusing probe defined for ContainerApps, since without grpc probe is same for both ContainerApps and ContainerAppsJobs